libraries/aws_vpc_endpoint.rb

```diff
@@ -0,0 +1,55 @@
+class AWSVpcEndpoint < Inspec.resource(1)
+  name 'aws_vpc_endpoint'
+  desc 'Test the configuration of a VPC endpoint.'
+  example "
+    describe aws_vpc_endpoint('vpce-1234') do
+      it { should exist }
+      its('status') { should eq 'available' }
+      its('vpc_id') { should eq 'vpc-00000' }
+      its('endpoint_type') { should eq 'Interface' }
+      its('service_name') { should eq 'com.amazonaws.ap-southeast-2.s3' }
+      its('route_tables') { should include 'rtb-00000' }
+    end
+  "
+
+  def initialize(vpc_endpoint_id)
+    @vpc_endpoint_id = vpc_endpoint_id
+    @vpc_endpoint = get_vpc_endpoint(vpc_endpoint_id)
+  end
+
+  def to_s
+    "VPC Endpoint #{@vpc_endpoint_id}"
+  end
+
+  def exists?
+    @vpc_endpoint != nil
+  end
+
+  def status
```

Comment:
Shouldn't it be "state"? (so it looks closer to the resource's attributes you want to test)